automation/.circleci/config.yml

```diff
@@ -0,0 +1,36 @@
+# CircleCI 2.0 configuration file.
+# https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: circleci/node:8.12.0-browsers
+      
+      # Specify service dependencies here if necessary
+      # CircleCI library of pre-built images documented at https://circleci.com/docs/2.0/circleci-images/
+
+    working_directory: ~/workbench/automation
+
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "package.json" }}
+          # fallback to using the latest cache if no exact match is found
+          - v1-dependencies-
+
+      - run: yarn install
+
+      - save_cache:
+          paths:
+            - node_modules
+          key: v1-dependencies-{{ checksum "package.json" }}
+     
+      # run tests 
+      - run: yarn run lint
+      - run: yarn run test
```

Comment:
Will this trigger off our builds/workflows in `.circleci`?

Reply:
I haven’t tested this out. It’s roughly a template for now